prefetch.bs

```diff
@@ -161,6 +161,10 @@ spec: no-vary-search; urlPrefix: https://httpwg.org/http-extensions/draft-ietf-h
     text: No-Vary-Search; url: name-http-header-field-definitio
 spec: resource-timing; urlPrefix: https://w3c.github.io/resource-timing/
   type: dfn; for: PerformanceResourceTiming; text: delivery type; url: dfn-delivery-type
+spec: WEBDRIVER; urlPrefix: https://w3c.github.io/webdriver/
+  type: dfn
+    text: error; url: dfn-errors
+    text: local end; url: dfn-local-ends
```

Comment:
These two lines should not be necessary; can you try deleting them and see what happens? I guess "error" is pretty generic, but disambiguation should be done via link-defaults not anchors.

Reply:
I'm unable to delete the two text fields because it errors with all anchors needing exactly one text. But I think you're right in that we don't need anchors as the link-defaults currently work for references, like in "Emit an event".